database/migration/src/main/resources/changesets/set_table_role.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog objectQuotingStrategy="QUOTE_ALL_OBJECTS"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <!-- The 'engineSharedCromwellDbRole' can be set via JAVA_OPTS if desired (eg -DengineSharedCromwellDbRole=...). -->
+    <property  name="engineSharedCromwellDbRole" value=""/>
```

Comment:
Note that the equivalent metadata property is called `sharedCromwellDbRole`. I want to name the two roles differently for the two databases, to reduce the change of one clobbering the other. 

I'd like to rename the metadata one to `metadataSharedCromwellDbRole`, and have added that option to the Helm chart as a step toward that migration.